### PR TITLE
Adding broker endpoints

### PR DESCRIPTION
**Pull request by jjmiranda**
## Summary
Adding delete and Create API Keys for subAccounts

## Additional Information
Using the sema nomenclature and stardards for this repo and OKX documentation.

### CODE REVIEW

File: src/rest-client.ts [hunk 1]
```diff
@@ -1354,4 +1354,30 @@ export class RestClient extends BaseRestClient {
...
+    return this.postPrivate('/api/v5/broker/nd/create-subaccount', params);
+  }
+
+  deleteSubAccount(params: {
+    subAcct:string,
+  }): Promise<any[]> {
+    return this.postPrivate('/api/v5/broker/nd/delete-subaccount', params);
+  } 
+
+  createAnAPIKeyForASubAccount (params: {
```
Comment (tiagosiebler) [inline src/rest-client.ts:1373]:
```suggestion
  createSubAccountAPIKey(params: {
```

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
Thank you for the PR! Small naming nitpick for the create api key fn. Could you also please revert the change to the package-lock.json? You can copy paste the one from the master branch, if you're not sure how to undo it. Thanks.

**Comment by tiagosiebler** [comment]
Please also increase the version number in the "package.json" file - it'll make the release process easier.

**Comment by jjmiranda** [comment]
Ready!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "okx-api",
-  "version": "1.2.0",
+  "version": "1.2.3",
```
Comment (tiagosiebler) [inline package.json:3]:
```suggestion
  "version": "1.2.1",
```

**Pull request merged**